### DISCUSSION TITLE
Firewall ufw on Debian.

### DISCUSSION THREAD

**Discussion by bonelifer** [jcorporation/myMPD#1182] [Q&A] [answered]
Setting up a touchscreen laptop as my myMPD server, with chromium setup as a kiosk for myMPD. Does this seem like a reasonable ufw setup. Are their other ports that need to be open in order for the system to work. Not opening it up to the internet, but someone might breach the WiFi.
```
# Set default policies to deny incoming and allow outgoing traffic
sudo ufw default deny incoming
sudo ufw default allow outgoing

# Enable the firewall (if not already enabled)
sudo ufw enable

# Allow SSH traffic (port 22)
sudo ufw allow ssh

# Allow HTTP traffic (port 80)
sudo ufw allow http

# Allow HTTPS traffic on a specific port (replace 8443 with the actual SSL port)
sudo ufw allow 443/tcp

# Allow MPD HTTP streaming (replace 8000 with the actual MPD HTTP port)
sudo ufw allow 8000/tcp

# Allow MyMPD access (replace 8080 with the actual MyMPD port)
sudo ufw allow 80/tcp

# Enable ufw (firewall)
sudo ufw enable
```

**Comment by jcorporation** [comment]
I think you do not need the two rules:

> sudo ufw allow 8000/tcp
> sudo ufw allow 80/tcp

80/tcp is the same as http.

**Comment by bonelifer** [comment]
Agreed on http and 80. used ChatGPT to create these. But isnt 8000 the http stream port for mpd.

**Reply by jcorporation** [reply, marked as answer]
If myMPD and MPD runs on the same host, myMPD connects through localhost to the MPD HTTP output. The browser connects to the port of myMPD and the myMPD daemon proxies it to MPD.
